parts/linux/cloud-init/artifacts/update_certs.sh

```diff
@@ -2,13 +2,18 @@
 set -uo pipefail
 
 certSource=/opt/certs
-certDestination=/usr/local/share/ca-certificates/certs
+certDestination="${1:-/usr/local/share/ca-certificates/certs}"
+updateCmd="${2:-update-ca-certificates -f}"
+destPrefix="aks-custom-"
 
```

Comment:
We should create certDestination dir here, as the new way of copying files from source will not create it automatically

Reply:
[ ! -d "$certDestination" ] && mkdir "$certDestination"

Reply:
```suggestion
[ ! -d "$certDestination" ] && mkdir "$certDestination"
```